src/Altinn.App.Api/Models/SigningAuthorizedOrganisationsResponse.cs

```diff
@@ -0,0 +1,33 @@
+namespace Altinn.App.Api.Models;
+
+/// <summary>
+/// Represents the response from the API when fetching organisations the user is authorized to sign on behalf of.
+/// </summary>
+public record SigningAuthorizedOrganisationsResponse
+{
+    /// <summary>
+    /// The list of authorised organisations.
+    /// </summary>
+    public required List<AuthorisedOrganisationDetails> Organisations { get; init; }
+}
+
+/// <summary>
+/// Represents the details of an authorized organisation.
+/// </summary>
+public record AuthorisedOrganisationDetails
```

Comment:
🇺🇸 ! 🙃  gjelder også flere steder og generelt i signering. Så kunne du kanskje sjekket for alle "Authorised" -> "Authorized" og "Organisation" -> "Organization" innen signering siden vi fortsatt kan gjøre sånne endringer? 😄 
```suggestion
public record AuthorizedOrganizationDetails
```

Reply:
Tok alle autorization. Organization var mye mer og treffer flere overflater så det får bli en separat pr